proj/test/dune

```diff
@@ -1,12 +1,9 @@
 (executable
  (name run_tests)
- (libraries
-  alcotest
-  test_sub1
-  test_sub2
-))
+ (libraries alcotest test_sub1 test_sub2))
```

Comment:
I like to have one thing per line so that the diff make sense when we add/remove items later.

Reply:
This was something that was auto-generated by ocamlformat and as we are going to use ocamlformat in future I though leaving the formatted code in would be better here